tests/crowdsourcing/tasks/test_chat_demo.py

```diff
@@ -339,16 +341,22 @@
 
 try:
 
-    # From the Mephisto repo
-    from examples.parlai_chat_task_demo.parlai_test_script import TASK_DIRECTORY
+    import mephisto
     from mephisto.server.blueprints.parlai_chat.parlai_chat_blueprint import (
         SharedParlAITaskState,
         BLUEPRINT_TYPE,
     )
 
-    from parlai.crowdsourcing.utils.tests import CrowdsourcingTestMixin
+    # From the Mephisto repo
+    mephisto_repo_folder = os.path.dirname(
+        os.path.dirname(os.path.abspath(mephisto.__file__))
+    )
+    sys.path.insert(1, mephisto_repo_folder)
```

Comment:
This is a huge red flag

Reply:
Agreed, but not sure about a short-term workaround given @JackUrb 's comment below. Maybe I can just add a TODO here for the time being?

Reply:
Hmm, I think this is a moot point, because if I change the order of the imports back to how it was before this PR, the test passes. I think this import issue will need to be addressed more robustly at some point, but it doesn't look to be a problem right now anymore